homeassistant/components/sensor/fail2ban.py

```diff
@@ -27,9 +26,11 @@
 DEFAULT_NAME = 'fail2ban'
 DEFAULT_LOG = '/var/log/fail2ban.log'
 SCAN_INTERVAL = timedelta(seconds=120)
+DEFAULT_UNITS = 'ip'
```

Comment:
This is not a unit, please remove.

Reply:
I agree, that is not a unit. The problem is that the value is numerical and unless a unit is specified, the graphic log will treat the value as text and cause heavy persormance issue when displaying it. I have yet to find a workaround and I am willing to remove this ugly hack as soon as this is fixed.